db/migrate/20220707151509_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.boolean :admin
+
```

Comment:
- [ ] I think for the user only name is enough, what do you think? I don't see the importance of adding the admin. I don't know if I'm right.